client/js/mousehandling.js

```diff
@@ -63,7 +67,7 @@ async function inputHandler(name, e) {
       if(ms.status == 'initial' || timeSinceStart < 250 && pixelsMoved < 10) {
         if(typeof jeEnabled == 'boolean' && jeEnabled)
           await jeClick(widgets.get(target.id));
-        else if(edit)
+        else if(editMovable)
```

Comment:
This check has noting to do with moving. Of course if jeEnabled, then this branch would not be reached, so there would be no this edit does not currently change any behavior.